eventuals/os.h

```diff
@@ -133,17 +125,19 @@ inline StackInfo GetStackInfo() {
       /* size = */ Bytes(size)};
 #endif // If Linux.
 
-#ifdef __MACH__
+#if defined(__MACH__) && (defined(__arm64__) || defined(__x86_64__))
   // For macOS `pthread_get_stackaddr_np` can return either the base
-  // or the end of the stack.
-  // Check 1274 line on the link below:
-  // https://android.googlesource.com/platform/art/+/master/runtime/thread.cc
+  // or the end of the stack. On Mac OS 10.7, it's the end.
```

Comment:
Please match the comments around you for consistency.
```suggestion
  // or the end of the stack. On macOS 10.7, it's the end.
```